vagrant/Vagrantfile

```diff
@@ -280,6 +280,36 @@ Vagrant.configure('2') do |config|
                            mount -t lustre /dev/mapper/mpathb /mnt/mdt0
                          SHELL
 
+        mds.vm.provision 'create-ldiskfs-lvm-fs',
+                         type: 'shell',
+                         run: 'never',
+                         path: 'scripts/create_ldiskfs_lvm_fs.sh',
+                         args: ['fs', '/dev/mapper/mpathb', 0, '/dev/mapper/mpatha']
+
+        mds.vm.provision 'mount-ldiskfs-lvm-fs',
+                         type: 'shell',
+                         run: 'never',
+                         inline: <<-SHELL
+                           mkdir -p /mnt/mgs
+                           mkdir -p /mnt/mdt0
+                           mount -t lustre /dev/mapper/mgt_vg-mgt /mnt/mgs
+                           mount -t lustre /dev/mapper/mdt0_vg-mdt /mnt/mdt0
+                         SHELL
+
+         mds.vm.provision 'ha-ldiskfs-lvm-fs-step2',
+                         type: 'shell',
+                         run: 'never',
+                         inline: <<-SHELL
+                           pcs cluster auth mds1.local mds2.local -u hacluster -p lustre
+                           pcs cluster setup --start --name mds-cluster mds1.local mds2.local --enable --token 17000
+                           pcs stonith create vboxfence fence_vbox ipaddr=10.0.2.2 login=root passwd=lustre
```

Comment:
We should require passwd as an environment variable when running this provisioner.

Reply:
How?